packages/eslint-config-shopify/rules/stylistic-issues.js

```diff
@@ -23,6 +23,8 @@ module.exports = {
   'func-names': 0,
   // Enforces use of function declarations or expressions
   'func-style': [1, 'declaration'],
+  // Blacklist certain identifiers to prevent them being used
+  'id-blacklist': [1, 'e'],
   // This option enforces minimum and maximum identifier lengths (variable names, property names etc.)
   'id-length': [1, {min: 2, properties: 'always', exceptions: ['x', 'y', 'i', 'j', '_']}],
```

Comment:
Could you add `$` to the exceptions so `import _ from 'jquery'` isn't problematic?